rbi/stdlib/reline.rbi

```diff
@@ -0,0 +1,223 @@
+# typed: __STDLIB_INTERNAL
+
+module Reline
+  FILENAME_COMPLETION_PROC = T.let(nil, T.nilable(Proc))
+  USERNAME_COMPLETION_PROC = T.let(nil, T.nilable(Proc))
+
+  class ConfigEncodingConversionError < StandardError; end
+
+  Key = Struct.new("Key", :char, :combined_char, :with_meta) do
+    def match?(other); end
+    alias_method :==, :match?
+  end
+  CursorPos = Struct.new(:x, :y)
+  DialogRenderInfo = Struct.new(
+    :pos,
+    :contents,
+    :bg_color,
+    :pointer_bg_color,
+    :fg_color,
+    :pointer_fg_color,
+    :width,
+    :height,
+    :scrollbar,
+    keyword_init: true
+  )
+
+  DIALOG_COLOR_APIS = [
+    :dialog_default_bg_color,
+    :dialog_default_bg_color_sequence,
+    :dialog_default_bg_color=,
+    :dialog_default_fg_color,
+    :dialog_default_fg_color_sequence,
+    :dialog_default_fg_color=,
+    :dialog_pointer_bg_color,
+    :dialog_pointer_bg_color_sequence,
+    :dialog_pointer_bg_color=,
+    :dialog_pointer_fg_color,
+    :dialog_pointer_fg_color_sequence,
+    :dialog_pointer_fg_color=
+  ]
+
+  class History < Array
+    def to_s; end
+
+    def delete_at(index); end
+
+    def [](index); end
+
+    def []=(index, val); end
+
+    def concat(*val); end
+
+    def push(*val); end
+
+    def <<(val); end
+  end
+
+  class Core
+    ATTR_READER_NAMES = [
+      :completion_append_character,
+      :basic_word_break_characters,
+      :completer_word_break_characters,
+      :basic_quote_characters,
+      :completer_quote_characters,
+      :filename_quote_characters,
+      :special_prefixes,
+      :completion_proc,
+      :output_modifier_proc,
+      :prompt_proc,
+      :auto_indent_proc,
+      :pre_input_hook,
+      :dig_perfect_match_proc,
+    ]
+
+    attr_reder :completion_append_character
```

Comment:
What is `attr_reder`?

Reply:
🤦 A typo. Sorry, I had to manually write this because I didn't sorbet would understand this line: https://github.com/ruby/reline/blob/b6ffa1714723b55a1e65134e49b2ed9da0560dc0/lib/reline.rb#L79